benchexec/tools/korn.py

```diff
@@ -34,8 +37,21 @@ def version(self, executable):
     def name(self):
         return "Korn"
 
-    def project_url(self):
-        return "https://github.com/gernst/korn"
```

Comment:
Let's keep this? Then you can remove it from the comment again.

Reply:
oh right